src/registry.rs

```diff
@@ -197,15 +197,15 @@ impl RegistryCore {
                     let mut pairs: Vec<proto::LabelPair> = hmap
                         .iter()
                         .map(|(k, v)| {
-                            let mut label = proto::LabelPair::new();
+                            let mut label = proto::LabelPair::default();
                             label.set_name(k.to_string());
                             label.set_value(v.to_string());
                             label
                         })
                         .collect();
 
                     for metric in m.mut_metric().iter_mut() {
-                        let mut labels = metric.take_label().into_vec();
+                        let mut labels: Vec<_> = metric.take_label().into();
```

Comment:
`Vec` doesn't implement `into_vec`, so I had to go around and use `Into::into`.

Reply:
How about just `let mut labels = metric.take_label();`

Reply:
That doesn't compile with the default features:

```
$ cargo build
   Compiling prometheus v0.6.0 (/Users/meqif/rust-prometheus)
error[E0599]: no method named `append` found for type `protobuf::RepeatedField<proto::LabelPair>` in the current scope
   --> src/registry.rs:209:32
    |
209 |                         labels.append(&mut pairs);
    |                                ^^^^^^

error: aborting due to previous error
```

Reply:
Oh. I got you.